app/views/grant_submissions/submissions/_form.html.haml

```diff
@@ -14,7 +14,13 @@
 .grid-x
   .cell.small-12
   .actions
-    = f.submit 'Submit', data: { disable_with: "Please wait..." }, class: 'button'
+    - if @submission.submitted?
+      = f.submit 'Save as Draft', class: 'secondary button', disabled: true
+      = f.submit 'Submit', class: 'success button', disabled: true
```

Comment:
I still like the idea of using button_tag here to pass the draft/submitted state. Avoids potential breaking of the controller b/c  the button text changes. Something like this:

`= button_tag "Save This as a Draft", type: 'submit', name: 'state', value: GrantSubmission::SUBMISSION_STATES[:draft] class:  'button btn-info'
= button_tag "Save This as a Draft", type: 'submit', name: 'state', value: GrantSubmission::SUBMISSION_STATES[:submitted] class:  'button btn-info'
`